spec/support/field_examples.rb

```diff
@@ -0,0 +1,33 @@
+module FieldExamples
+  # an example calling this is expected to provided an instance of a
+  # DocumentType::*Field
+  #
+  # For example:
+  #   it_behaves_like "document type field", DocumentType::PrimaryPublishingOrganisationField.new
+  #
+  shared_examples "document type field" do |field|
```

Comment:
Rather than passing in the field object this looks like one of the rare cases where the one-liner syntax for RSpec is probably most applicable. 

In that instance it'd be like

```
shared_examples "document type field" do
  it { is_expected.to respond_to(:id).with(0).arguments }
  # etc
end

# and called like:


RSpec.describe DocumentType::PrimaryPublishingOrganisationField do
  it_behaves_like "document type field"
```

The disadvantage of passing the initialised object in, as you've done, is that you end up with the same object under test for each test. Not too big a deal given the scenario but is something of a smell.